pkg/syncthing/config.go

```diff
@@ -0,0 +1,52 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+package syncthing
+
+import (
+	"encoding/xml"
+
+	"github.com/syncthing/syncthing/lib/config"
+)
+
+// @source: https://docs.syncthing.net/users/config.html
+func InitConfig() *config.Configuration {
+	return &config.Configuration{
+		Version: 37,
+		GUI: config.GUIConfiguration{
+			Enabled:    true,
+			RawAddress: "127.0.0.1:8384",
```

Comment:
Where will this addr be used? Do we need to configure it to other address e.g. 0.0.0.0?

Reply:
This address will be accessed by the local envd client via port forwarding.